### PR TITLE
Fix https://github.com/ant-media/Ant-Media-Server/issues/2490

### PR DESCRIPTION
**Pull request by mekya**

### CODE REVIEW

File: src/main/webapp/js/webrtc_adaptor.js [hunk 1]
```diff
@@ -349,11 +352,13 @@ function WebRTCAdaptor(initialValues)
...
 			track.stop();
 		});
+		
+		
 
 	}
 
 	/**
-	 * Checks browser supports screen share feature
+	 * Checks broenwser supports screen share feature
```
Comment (eneskuluk) [inline src/main/webapp/js/webrtc_adaptor.js:361]: there is no need to change this comment line

**Review by eneskuluk — COMMENTED** [review]
fixing the comment would be good

File: src/main/webapp/js/webrtc_adaptor.js (hunk 1 above)
Comment (mekya) [reply src/main/webapp/js/webrtc_adaptor.js]: Ok. It's fixed. Take a look at the latest commit

**Review by eneskuluk — APPROVED** [review]
all in order

**Pull request merged**